views/auth/register.handlebars

```diff
@@ -0,0 +1,125 @@
+<div class="pb-3">
+  <div>Already have an account? <a href="/login">Login here</a></div>
+</div>
+
+<form action='/register' method="POST">
+    <div class="form-row">
+        <div class="col">
+            <label for="firstName">First Name</label>
+            <input type="text" class="form-control" name="firstName" value="{{firstName}}" placeholder="First name" required>
+        </div>
+        <div class="col">
+            <label for="lastName">Last Name</label>
+            <input type="text" class="form-control" name="lastName" value="{{lastName}}" placeholder="Last name" required>
+        </div>
+    </div>
+    <div class="form-row">
+        <div class="col">
+            <label for="Email">Email</label>
+            <input type="email" class="form-control" name="email" value="{{email}}" placeholder="Email" required>
+        </div>
+    </div>
+    <div class="form-row">
+        <div class="col">
+            <label for="password">Password</label>
+            <input type="password" class="form-control" name="password" value="{{password}}" placeholder="Password" required>
+        </div>
+        <div class="col">
+            <label for="confirmPassword">Confirm password</label>
+            <input type="password" class="form-control" name="confirmPassword" value="{{confirmPassword}}" placeholder="Confirm password" required>
+        </div>
+    </div>
+    <div class="form-row">
+        <div class="col">
+            <label for="Address">Address</label>
+            <input type="text" class="form-control" name="address" value="{{address}}" placeholder="1343 Anywhere St Apt 100" required>
+        </div>
+    </div>
+    <div class="form-row">
+        <div class="col">
+            <label for="city"> City</label>
+            <input type="text" class="form-control" name="city" value="{{city}}" required>
+        </div>
+        <div class="col">
+            <label for="state">State</label>
```

Comment:
I'd probably make state/province a free form field and create a dropdown for countries. I'd even probably create a Countries table to list the codes and display values and pull from there to populate the dropdown.

Check out this example! https://bootsnipp.com/snippets/r8W

Reply:
Changed state to freeform and added the country dropdown. I decided to put the dropdown in HTML for now, mainly for the sake of time so you can get to test your methods